### PR TITLE
docs(en): update endpoints info

### PR DESCRIPTION
**Pull request by gao-sun**
## Summary

update endpoints info per our new version

## Testing

locally

### CODE REVIEW

File: docs/docs/recipes/deployment/README.md [hunk 1]
```diff
@@ -14,7 +14,16 @@ You can set env directly or put a `.env` file inside the Logto project root. If
...
+- `ENDPOINT` You may specify a URL with your custom domain for production (E.g. `ENDPOINT=https://logto.domain.com`). This will also affect the value of the [OIDC issuer identifier](https://openid.net/specs/openid-connect-core-1_0.html#IssuerIdentifier).
+
+**Enable Admin Console**
+
+- `ADMIN_PORT` The port that Logto Admin Console listens to. Default `3002`.
+- `ADMIN_ENDPOINT` You may specify a URL with your custom domain for production (E.g. `ADMIN_ENDPOINT=https://admin.domain.com`). This will also affect the value of Admin Console Redirect URIs.
+
+**Disable Admin Console**
+
+- `ADMIN_DISABLE_LOCALHOST` Set it to `1` or `true` to close the port for Admin Console. With `ADMIN_ENDPOINT` unset, it'll completely close the Admin Console.
```
Comment (charIeszhao) [inline docs/docs/recipes/deployment/README.md:26]:
```suggestion
- `ADMIN_DISABLE_LOCALHOST` Set it to `1` or `true` to close the port for Admin Console. With `ADMIN_ENDPOINT` unset, it'll completely disable the Admin Console.
```
Use `disable` or `turn off` would be better than `close` I think

File: docs/docs/references/core/README.mdx [hunk 2]
```diff
@@ -102,6 +102,14 @@ We provide a standard OIDC [Userinfo Endpoint](https://openid.net/specs/openid-c
 
 <BearerAuth />
 
+### About refresh token reuse interval
+
+You may notice Refresh Token has a rotation delay (3s). This delay helps to avoid concurrency issues when exchanging the rotating refresh token multiple times within a given timeframe.
```
Comment (charIeszhao) [inline docs/docs/references/core/README.mdx:107]:
```suggestion
You may notice Refresh Token has a rotation delay (3s). This delay helps avoid concurrency issues when exchanging the rotating refresh token multiple times within a given timeframe.
```

File: docs/docs/references/core/README.mdx [hunk 3]
```diff
@@ -102,6 +102,14 @@ We provide a standard OIDC [Userinfo Endpoint](https://openid.net/specs/openid-c
...
+
+During the leeway window (in seconds), the consumed refresh token will be considered as valid.
```
Comment (charIeszhao) [inline docs/docs/references/core/README.mdx:109]:
```suggestion
During the leeway window (in seconds), the consumed refresh token will still be considered as valid.
```

File: docs/docs/references/core/configuration.md [hunk 4]
```diff
@@ -19,15 +19,18 @@ If you run Logto via `npm start` in the project root, `NODE_ENV` will always be
...
+| TRUST_PROXY_HEADER      | `false`                              | `boolean`                                                | Ditto.                                                                                                                                                                                                                   |
+| ENDPOINT                | `'protocol://localhost:$PORT'`       | `string`                                                 | You may specify a URL with your custom domain for online testing or production. This will also affect the value of the [OIDC issuer identifier](https://openid.net/specs/openid-connect-core-1_0.html#IssuerIdentifier). |
+| ADMIN_ENDPOINT          | `'protocol://localhost:$ADMIN_PORT'` | `string`                                                 | You may specify a URL with your custom domain for production (E.g. `ADMIN_ENDPOINT=https://admin.domain.com`). This will also affect the value of Admin Console Redirect URIs.                                           |
```
Comment (charIeszhao) [inline docs/docs/references/core/configuration.md:33]: Maybe we should also add `DISABLE_ADMIN_LOCALHOST`?

Comment (gao-sun) [reply docs/docs/references/core/configuration.md:33]: added bruh

**Review by charIeszhao — APPROVED** [review]
LGTM bruh

**Pull request merged**